リスト機能実装（未完成）

LLCCLL opened the pull request:
データベースからデータを読んで、リストの形で出力する機能を実装完了しました。
(今日basis_springリポジトリにpushしたコミットのコードも多数参考しました)
ですが、コード自体は全部完成したものの、出力過程でまだ問題が残っています。
詳しいことはコミットの内容をご参考ください。

Commit:
add register action

btijnb committed:
modify registerController

Commit:
Modify function to show list (not perfect)

1) To use this file correctly,
The source codes below must be added Servers/context.xml in STS.

     <Resource auth="Container"
    			  driverClassName="com.mysql.jdbc.Driver"
    			  maxActive="50"
    			  maxWait="1000"
    			  name="jdbc/ums"
    			  password="spring"
    			  type="javax.sql.DataSource"
    			  url="jdbc:mysql://localhost:3306/ums"
    			  username="root"/>

2) There is an error message saying 'Resource Not Found Error!' on view.

btijnb commented on ums/src/main/java/jp/co/asia/archive/ums/domain/repository/UmsDAO.java:
```diff
@@ -0,0 +1,69 @@
+import jp.co.asia.archive.ums.domain.model.UVO;
+
+public class UmsDAO {
+
```
今度説明しますが、mybatisを使用したサンプルを作成しますので参考して修正してください。

LLCCLL replied:
かしこまりました。

Commit:
Modify form and list

1) It works, but not match program specification yet

2) To use this file correctly,
The source codes below must be added Servers/context.xml in STS.

     <Resource auth="Container"
    			  driverClassName="com.mysql.jdbc.Driver"
    			  maxActive="50"
    			  maxWait="1000"
    			  name="jdbc/ums"
    			  password="spring"
    			  type="javax.sql.DataSource"
    			  url="jdbc:mysql://localhost:3306/ums"
    			  username="root"/>

Commit:
Add views and controllers

still have some issues

Commit:
Solve issue that is about parameters

However, still have some issues: post method can't be used.

Commit:
Add views and controllers

of update and delete function.
detailed codes are still incompleted.

Commit:
git pull origin/viewActv

Commit:
Apply JDBC Template

Commit:
Merge branch 'master' of https://github.com/btijnb/userMng

Commit:
Apply annotations
